meta/main.yml

```diff
@@ -4,7 +4,7 @@ galaxy_info:
   description: Ansible role for configuring Logging system (e.g. rsyslog)
   galaxy_tags: ['logging', 'rsyslog', 'redhat', 'fedora', 'centos']
   company: Red Hat, Inc.
-  license: GPL-3.0+
+  license: GPL-3.0-only,MIT
```

Comment:
Is this the correct syntax for specifying multiple licenses in the meta/main.yml?

Reply:
no help here :-( https://galaxy.ansible.com/docs/contributing/creating_role.html#role-metadata

Reply:
for kernel_settings I used `license: GPLv2+,MIT`

Reply:
Ok, do you think `license: GPLv3,MIT` is better?

Reply:
I think `license: GPL-3.0-only,MIT` is fine.
* it is unambiguous, given familiarity with spdx and fedora license name/abbreviations
* the format of the meta/main.yml license field isn't documented
* galaxy ui doesn't use it